build/resources/main/schema.sql

```diff
@@ -1,13 +1,17 @@
-CREATE TABLE dog_breed (
-    id VARCHAR(255) PRIMARY KEY,
-    name VARCHAR(255) NOT NULL,
-    sub_breeds TEXT
+CREATE TABLE dogBreeds(
```

Comment:
according to Flyway spec https://www.red-gate.com/blog/database-devops/flyway-naming-patterns-matter  sql scripts with migrations must be in the resources folder and follow specific naming pattern.

Reply:
added